readme.md

```diff
@@ -214,14 +234,51 @@ There's no right way to do user stories. Their purpose is:
 
 If your method of writing user stories isn't doing any of those, do it differently.
 
-## You Do: User Stories for [Final Countdown](https://github.com/ga-wdi-exercises/final-countdown) (20 / 110 min)
+## Not the Olympics - Bronze/Silver/Gold
+In the Olympics, we strive for gold. In agile software devlopment, we strive for the bronze in order to reach the gold. That is not to say we set low expectations for ourselves. But rather, the gold should be just be an extension of the bronze achievement.
 
-Without looking at the prompt for Final Countdown, create user stories for it and convert them to code stories.
+### MVP - Minimum Viable Product
+The version of a new product which allows a team to collect the maximum amount of validated learning about customers with the least effort.
 
-## You Do: User Stories for ATM (20 / 130 min)
+[Minimum Viable Product](https://www.youtube.com/watch?v=1FoCbbbcYT8)
+
+In the context of the WDI student, MVP is a bit different, but there are a lot of things that correlate. Create a product that does what the prompt asks you. Whatever that takes, get the thing to work the way you intended first. Then add features and other cool things!
+
+### Bronze - Your MVP
+- The bare necessities to be functional and meet requirements.
+  - In WDI: does it meet the criteria for the project?
+  - In the real world: does it meet the business requirements in order to start an effective feedback loop
+
+### Silver
+- Improve user experience
+  - in WDI: one cool feature! Enhancing what you learned in class
+  - in the real world: What can we push to the next iteration? Where can we add value?
+
+### Gold
+- Nice to haves
+  - in WDI: push beyond what you've learned in class
+  - in the real world: next steps to maximize ROI
+
+> In the real world, often things that are gold transition into their own subsets of bronze/silver/gold
+
+In WDI, we're trying to get that MVP during project week as soon as possible. So that we can start to think about improvements and polishing. That is not to say that we should rush the bronze. Make sure that the code is well written, maintainable, and functional before thinking about moving to the Silver implementation.
 
+When we start to write user stories, break these user stories in these 3 categories.
+
+## You Do: User Stories for ATM (20 / 130 min)
 [ATM application](https://github.com/ga-wdi-exercises/atm)
 
+## Wireframing
+A wireframe is a simple blueprint/template/sketch/visual outline of the components of your website. Though there are some great digital wireframing tools out there, all you really need to make a wireframe for your personal use is a piece of paper and something to write with.
+
+- Curious to learn more about wireframes? Ask a UXDI student!
```

Comment:
Maybe include some references to guides on wire framing, tools that they can use for free (e.g. [this list](http://mashable.com/2010/07/15/wireframing-tools/#hFSISJWSsgqX) )